### ISSUE TITLE
Homebridge-musiccast-multiroom might not be ready for Homebridge v2.0

### ISSUE THREAD

**Issue by CezaryCiosek** [cgierke/homebridge-musiccast#12]
![Zrzut ekranu 2024-12-17 o 15 27 09](https://github.com/user-attachments/assets/0095f880-7b27-4fca-81e7-a1feb2f0aeef)

Is it possible that you will mark/fix it?

**Comment by cgierke** [comment]
Good point - I think it will be compatible without any changes, but I couldn't test it yet. Are you maybe running Homebridge 2 already?

**Comment by CezaryCiosek** [comment]
not yet but maybe today or right after holidays :)

**Comment by CezaryCiosek** [comment]
Hi @cgierke, I just tested plugin on newest version of hb 2.0 beta and it look that everything works like a charm ;) I think you can mark it as a `2.0 ready` ;)

**Comment by cgierke** [comment]
Thanks a lot @CezaryCiosek, I created a Homebridge 2.0 compatible release today!
